tailwind.config.js

```diff
@@ -6,6 +6,12 @@ module.exports = {
   ],
   darkMode: "class",
   theme: {
+    borderWidth: {
+      DEFAULT : '1px',
+      '0': '0',
+      '2' : '2px',
+      '3' : '3px',
+    },
```

Comment:
Are they changes needed? 🤔

Reply:
border-l-3 is not handled by tailwind directly, that's why I've changed it. But instead of border-l-3 if we add border-l-2 , then it's not needed. What do you think @eddiejaoude ?

Reply:
Using the [arbitrary value](https://tailwindcss.com/docs/border-width#arbitrary-values) feature of tailwind might be a better idea

Reply:
yes, I think it might be better. Thank you @dan-mba